user_management/api/tests/test_throttling.py

```diff
@@ -77,6 +77,13 @@ def test_user_auth_throttle_username(self):
         response = self.view_class.as_view()(request)
         self.assertEqual(response.status_code, self.throttle_expected_status)
 
+    def test_authenticated_user_not_throttled(self):
+        """An already authenticated user is not throttled."""
+        request = self.create_request('post', data={})
+        view = self.view_class.as_view()
+        response = view(request)
+        self.assertNotEqual(response.status_code, self.throttle_expected_status)
```

Comment:
Do you need to decorate the function with `@patch(THROTTLE_RATE_PATH, new={'logins': '1/minute'})` and call it two times to see if the authenticated user is not throttled?